clients/cobol-lsp-vscode-extension/src/services/CopybooksDownloader.ts

```diff
@@ -103,10 +113,12 @@ export class CopybooksDownloader implements vscode.Disposable {
         this.queue.stop();
     }
 
+    //TODO: NO INVOKED IN THE CODE BUT USED BY UNIT TEST
+    //TODO: VERIFY ZOWE ERRORS ARE STILL SUPPORTED
```

Comment:
@zacanbrcom remove this comment and verify that handleQueue() is correctly engaged from the ```start()``` method